doc/source/whatsnew/v0.24.2.rst

```diff
@@ -50,6 +50,7 @@ Bug Fixes
 **I/O**
 
 - Bug in reading a HDF5 table-format ``DataFrame`` created in Python 2, in Python 3 (:issue:`24925`)
+- KeyError raised by :func:`parse_table_schema` when reading a JSON with ``orient='table'`` generated by :meth:`DataFrame.to_json` with ``index=False``
```

Comment:
Bug in .......

you don't need to say KeyError, nor mention the parse_table_schema which is internal

add the issue number here

Reply:
Done @jreback